WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/PostsAndPagesUseCaseTest.kt

```diff
@@ -31,20 +31,31 @@ import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Empty
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Link
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ListItemWithIcon
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Title
+import org.wordpress.android.ui.stats.refresh.lists.sections.granular.SelectedDateProvider
 import org.wordpress.android.ui.stats.refresh.utils.StatsDateFormatter
 import org.wordpress.android.ui.stats.refresh.utils.toFormattedString
+import java.util.Date
 
 private const val pageSize = 6
 private val statsGranularity = DAYS
+private val currentDate = Date(10)
```

Comment:
I'd consider renaming this value as its name is a bit confusing.

Off-topic: Out of curiosity - why do you pass `10` to the constructor? At first I thought you want to test it with a non-default date, however adding 10ms doesn't change the date (just the time) right?

Reply:
`Date()` is current date which changes every time you run the test, `Date(int)` is a 1970 basic date + int ms. However, I should be using just `Date(0)`, 10 doesn't make any sense. I think we should be using constant values in tests.